src/web/app.js

```diff
@@ -49,21 +51,22 @@ export default function webapp(web: Web) {
     }
 
     const { id } = socket.request.session.passport.user;
-    if (!base.bot.server.members.has(id)) {
+    const currentServer = base.bot.server;
```

Comment:
No need for different name (just destructure; you could also be pulling out `voiceChannel` here.)